views/components/Graphs/GraphTable.jsx

```diff
@@ -0,0 +1,59 @@
+import React from 'react'
+import {
+  TableBody,
+  TableCell,
+  TableHead,
+  TableRow,
+  Table,
+  Paper,
+} from '@material-ui/core'
+
+const GraphTable = ({ graph }) => {
+  return (
+    <Paper>
+      <Table>
+        <TableHead>
+          <TableRow>
+            <TableCell align="center">Site</TableCell>
+            {graph?.labels?.map(({ name }) => (
```

Comment:
I think it's worth putting an explicit return here if we don't have a graph so we don't need to safely navigate the graph object everywhere.

```js
if (!graph) {
  return null
}
```

```suggestion
            {graph.labels?.map(({ name }) => (
```

Reply:
https://github.com/AMP-SCZ/dpdash/pull/175/commits/6eaaa4b6b4bfc4375ce6e4251bede79d0598f08b

Reply:
the optional chaining part was missing 
https://github.com/AMP-SCZ/dpdash/pull/175/commits/ac6014d28b1ede4b997c0887a70a0f731972be07